common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -0,0 +1,22 @@
+package navigation
+
+import conf.Configuration.id.membershipUrl
+
+object UrlHelpers {
+
+  def getJobUrl(editionId: String): String = {
+    if (editionId == "au") {
+      "https://jobs.theguardian.com/landingpage/2868291/jobs-australia-html/?INTCMP=jobs_au_web_newheader"
+    } else {
+      s"https://jobs.theguardian.com?INTCMP=jobs_${editionId}_web_newheader"
+    }
+  }
+
+  def getContributionOrSupporterUrl(editionId: String): String = {
```

Comment:
I didn't add any tests because this is simple enough, but maybe I should... what do you think?